mandatory/1-syntax-errors.js

```diff
@@ -1,17 +1,27 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
+let x = 3;
+let y = 4;
+let z = 6;
+function addNumbers(x, y, z) {
+  return x + y + z;
+}
+console.log(addNumbers);
+
 
-function addNumbers(a b c) {
-  return a + b + c;
+let name = "Sonjide";
+let age = 27;
+function introduceMe(name, age){
+  return `Hello, my name is ${name} and I am ${age} years old`;
```

Comment:
Well done. But As I said earlier, not necessary to add variables and console.log.

Reply:
Thank you for your help! My apologies I misunderstood the assignment. Resolved and updated :)